packages/wdio-cli/src/commands/repl.js

```diff
@@ -1,12 +1,12 @@
 import { remote } from 'webdriverio'
 import { hasWdioSyncSupport } from '@wdio/utils'
+import { getCapabilities } from '../utils'
 
-export const command = 'repl <browserName>'
+export const command = 'repl <option> [capabilities]'
```

Comment:
Could you add the `capabilities` with the available flags to the yargs builder function so this way it can be displayed in the `--help` flag? You could probably use yarg's [positional](https://github.com/yargs/yargs/blob/master/docs/advanced.md#describing-positional-arguments) api for this.

Reply:
Added builder for wdio repl --help